src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRentalCost = 40;
+  let total = days * dailyRentalCost;
```

Comment:
Consider using a more descriptive variable name instead of `total`. For example, `rentalCost` would be more informative. 😊